pypeit/core/telluric.py

```diff
@@ -172,56 +183,48 @@ def read_telluric_grid(filename, wave_min=None, wave_max=None, pad_frac = 0.10):
     dwave, dloglam, resln_guess, pix_per_sigma = wvutils.get_sampling(wave_grid)
     tell_pad_pix = int(np.ceil(10.0 * pix_per_sigma))
 
-    tell_dict = dict(wave_grid=wave_grid, dloglam=dloglam,
-                     resln_guess=resln_guess, pix_per_sigma=pix_per_sigma, tell_pad_pix=tell_pad_pix,
-                     pressure_grid=pg, temp_grid=tg, h2o_grid=hg, airmass_grid=ag, tell_grid=model_grid)
-    return tell_dict
+    return dict(wave_grid=wave_grid, dloglam=dloglam, resln_guess=resln_guess,
```

Comment:
More elegant would be an object here instead of a dict. But it need not be a data container. Anyway, it is not urgent.

Reply:
This is inherited from the previous code.